src/NetAnalyzers/CSharp/Microsoft.NetCore.Analyzers/Performance/CSharpUseSearchValues.cs

```diff
@@ -14,6 +14,12 @@ namespace Microsoft.NetCore.CSharp.Analyzers.Performance
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public sealed class CSharpUseSearchValuesAnalyzer : UseSearchValuesAnalyzer
     {
+        // The referenced SDK version doesn't yet contain these SyntaxKind values
+        // https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/Syntax/SyntaxKind.cs
+        private const SyntaxKind Utf8StringLiteralToken = (SyntaxKind)8520;
+        private const SyntaxKind Utf8StringLiteralExpression = (SyntaxKind)8756;
+        private const SyntaxKind CollectionExpression = (SyntaxKind)9076;
```

Comment:
Please move these to the lightup layer: https://github.com/dotnet/roslyn-analyzers/tree/main/src/Utilities/Compiler/Lightup. Needs a new file for SyntaxKinds, like exists for OperationKinds.